src/components/Calculator.js

```diff
@@ -29,41 +20,40 @@ render() {
         </div>
```

Comment:
- [ ] <b>REQUIRED:</b> There is a problem in displaying, the `operator ` must come between the `total` and `next` so that it doesn't function the way it is functioning in this [video](https://www.loom.com/share/05e976e4cc5c43af9b2c57705c6f23a6).
- [ ] <b>REQUIRED:</b> And also, please make sure that clear all functionality does not clear everything, instead, it should leave `0` on the screen.

Reply:
I have fixed the issue, The operator is now between the total and next

The calculator is no longer returning null